Vagrantfile

```diff
@@ -17,8 +17,8 @@ Vagrant.configure("2") do |config|
 
   config.vm.box = "bento/ubuntu-20.04"
 
-  config.vm.synced_folder ".", "/vagrant", disabled: true
-  config.vm.synced_folder ".", "/srv/zulip"
+  config.vm.synced_folder ".", "/vargant", disabled: true, docker_consistency: "z"
```

Comment:
There’s no need to change any options for a disabled folder.

Reply:
Changed it. Why is there a disabled bind mount anyway?

Reply:
The line is intended to disable the mount that Vagrant enables by default, so we can move it elsewhere.

You’ve misspelled `vagrant` this time.

Reply:
Oh no - fixed. (Don't know how this happened...)

What would be a more appropriate location for disabling the mount?